packages/interface/package.json

```
@@ -0,0 +1,37 @@
+{
+	"name": "@wordpress/interface",
+	"version": "0.0.1",
+	"private": true,
+	"description": "Interface module for WordPress.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/interface/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git",
+		"directory": "packages/interface"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"main": "build/index.js",
+	"module": "build-module/index.js",
+	"react-native": "src/index",
+	"dependencies": {
+		"@babel/runtime": "^7.8.3",
+		"@wordpress/components": "../components",
+		"@wordpress/data": "4.14.0",
+		"@wordpress/element": "../element",
+		"@wordpress/i18n": "../i18n",
+		"@wordpress/icons": "../icons",
+		"@wordpress/plugins": "../plugins",
```

Dependencies should be declared as follows:

```suggestion
		"@wordpress/components": "file:../components",
		"@wordpress/data": "file:../data",
		"@wordpress/element": "file:../element",
		"@wordpress/i18n": "file:../i18n",
		"@wordpress/icons": "file:../icons",
		"@wordpress/plugins": "file:../plugins",
```